src/components/ExerciseNavigation.tsx

```diff
@@ -0,0 +1,53 @@
+import ExerciseLink from "@/components/ExerciseLink";
+
+const exerciseLinksData = [
+  { slug: "/space/intro", text: "Intro & Overview", imageSrc: "/intro.png" },
+  {
+    slug: "/space/password",
+    text: "Passwort Sicherheit",
+    imageSrc: "/passwort.png",
+  },
+  {
+    slug: "/space/privatsphäre",
+    text: "Privatsphäre",
+    imageSrc: "/privacy.png",
+  },
+  {
+    slug: "/space/daten-verarbeitung",
+    text: "Daten Verarbeitung",
+    imageSrc: "/data-processing.png",
+  },
+  { slug: "/space/phishing", text: "Phishing", imageSrc: "/phishing.png" },
+  { slug: "/space/rechte", text: "Meine Rechte", imageSrc: "/rights.png" },
+];
+
+export function ExerciseNavigation() {
+  return (
```

Comment:
we need to rearrange this by using a grid to support dynamic display sizes, but i will do this in the follow up ticket